02/app.js

```diff
@@ -1 +1,24 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+const uList = document.querySelector("ul");
+const search = uList.querySelectorAll('a[data-url]');
+
+console.log(search);
+const a = search[0].dataset.url;
+const b = search[1].dataset.url;
+const c = search[2].dataset.url;
+console.log(a);
+console.log(b);
+console.log(c);
+const aFirst = document.getElementsByTagName("a")[0].setAttribute("href", a);
+const aSecond = document.getElementsByTagName("a")[1].setAttribute("href", b);
+const aThird = document.getElementsByTagName("a")[2].setAttribute("href", c);
```

Comment:
Spróbuj to zrobić wykorzystując pętle `for`. To co jest niżej nie do końca temu odpowiada.